src/filelogger.jl

```diff
@@ -3,11 +3,14 @@ struct FileLogger <: AbstractLogger
     always_flush::Bool
 end
 
-function FileLogger(path; append=false, always_flush=true)
+function FileLogger(path::AbstractString; append=false, kwargs...)

```

Comment:
we don't need to restrict this, and adding the restriction breaks FilePathBase compatibility

```suggestion
function FileLogger(path; append=false, kwargs...)
```